ompi/mpi/fortran/base/fint_2_int.h

```diff
@@ -163,7 +164,29 @@
  * Define MACROS to take account of different size of logical from int
  */
 
-#if OMPI_SIZEOF_FORTRAN_LOGICAL == SIZEOF_INT
```

Comment:
@markalle I believe you do not need this any more.
I added some tests in open-mpi/ompi-tests#104, built both Open MPI and the ibm test suite with `-fdefault-integer-8` and had no issue with the stock/current `ompi/mpi/fortran/base/fint_2_int.h`

Reply:
Oh gosh, this is embarrassing. I agree. My local tree is changed like you described but apparently in the thing I pushed I failed to "git add" that file. Let me re-push